web-animations-1/Overview.bs

```diff
@@ -3301,24 +3307,24 @@ a <a>target element</a> for which computed property values can be calculated.
         <var>computed keyframes</var>.
 
     1.  For each property specified in <var>keyframe</var>, calculate the
-        computed value specified on <var>keyframe</var> using the <a>target
-        element</a> as the context for computing values and add the
+        computed value specified on <var>keyframe</var> using the <a>effect target</a>
+        as the context for computing values and add the
         corresponding property and computed value to <var>computed
         keyframe</var>.
         For shorthand properties, add the equivalent longhand properties.
         For logical properties [[!CSS-LOGICAL-1]], add the [=equivalent physical
         properties=] [[!CSS-WRITING-MODES-4]] based on the computed value of
-        'writing-mode' and/or 'direction' for <a>target element</a>.
+        'writing-mode' and/or 'direction' for the <a>effect target</a>.
 
-        For example, if <var>keyframe</var> has a value of &lsquo;12pt&rsquo;
+        For example, if <var>keyframe</var> has a value of ‘12pt’
         for the 'border-width' property, the user agent may produce a computed
-        value of &lsquo;16px&rsquo; for each of the longhand properties:
+        value of ‘16px’ for each of the longhand properties:
```

Comment:
Should these changes be a separate patch?

Reply:
Perhaps you're right and this and the previous change should go in a separate cleanup patch.